doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,138 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+Lots of GitHub issues and pull requests are being created daily.
+To help us manage them we have “labels”:
+to classify them by area, type, priority, or to mark them in need of attention.
+
+All GitHub issues or pull requests must be appropriately labeled.
+Issues and PRs often have multiple labels assigned,
+to help classify them in the different available categories.
+When reviewing a PR, if it has missing or incorrect labels, maintainers shall
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, reduces the changes of duplicates, etc.
```

Comment:
I'm having a little trouble understanding what "reduces the changes of duplicates" means -- do you think you can clarify this?

Reply:
would it be enough to say "reduces the changes of duplicate issue reports"?
(it is just a quick motivational paragraph, I would not it want to become too long)

Reply:
OK, so this is about avoiding dupes? How about "avoids duplicate issue reports"?